backend/app.py

```diff
@@ -172,6 +177,47 @@ class CsetsUpdate(BaseModel):
     row_index_data_map: Dict[int, Dict[str, Any]] = {}
 
 
+@app.get("/concept-set-names")
+@app.get("/datasets/csets/names")
+@app.get("/jq-cset-names")
+def cset_names() -> Union[Dict, List]:
+    """Get concept set names"""
+    return csets_read(field_filter=['conceptSetNameOMOP'])
+
+
+@app.get("/datasets/csets")
+def csets_read(field_filter: Union[List[str], None] = Query(default=None)) -> Union[Dict, List]:
+    """Get concept sets
+
+    field_filter: If present, the data returned will only contain these fields. Example: Passing `conceptSetNameOMOP` as
+    the only field_filter for OMOPConceptSet will return a string list of concept set names.
+
+    Resources: jq docs: https://stedolan.github.io/jq/manual/ , jq python api doc: https://github.com/mwilliamson/jq.py
+    """
+    if field_filter:
+        if len(field_filter) > 1:
+            d = {'error': 'Currently only 1 field_filter is allowed.'}
```

Comment:
@Sigfried Not that we need more than one `field_filter` right now; we just need `field_filter= conceptSetNameOMOP` for our current purposes. But in case a user passes multiple `field_filter`s, I return this error.